pkg/collector/corechecks/system/memory_windows.go

```diff
@@ -2,19 +2,33 @@
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2018 Datadog, Inc.
-
 // +build windows
 
 package system
 
 import (
 	"fmt"
+	"runtime"
 
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
 	log "github.com/cihub/seelog"
 
 	"github.com/DataDog/datadog-agent/pkg/aggregator"
 )
 
+// For testing purpose
+var virtualMemory = winutil.VirtualMemory
+var swapMemory = winutil.SwapMemory
+var runtimeOS = runtime.GOOS
+
+// MemoryCheck doesn't need additional fields
```

Comment:
`MemoryCheck` and `mbSize` are the same for linux and windows, we should leave then in the commong `memory.go`.